lib/shared/store.js

```diff
@@ -212,9 +213,12 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
-      if (_.isNil(action.data.speed)) {
+      if (_.some([
```

Comment:
I think this would be more readable without using `_.some`, especially as you're already using other logical-operators below.